opensuse-tomcat-image/src/main/docker/web.xml

```diff
@@ -0,0 +1,4753 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+
+    Copyright 2017-2022 Micro Focus or one of its affiliates.
+
+    Licensed under the Apache License, Version 2.0 (the "License");
+    you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+         http://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+    distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+    limitations under the License.
+
+-->
+<web-app xmlns="http://xmlns.jcp.org/xml/ns/javaee"
```

Comment:
Yes it is from https://github.com/apache/tomcat/blob/main/conf/web.xml so it just overwrites some parameters in here to increase the security.

Reply:
Ok in that case I'd be very uncertain about us adding a Micro Focus copyright to it.  We only wrote a small portion of it so adding our copyright may not be legal.  But it doesn't matter anyway because I want you to remove it.

The problem is that there is no way to see what changes we're actually making to the file.  This needs to be adjusted so that instead of checking in the updated file verbatim, that we create a web.xslt transform, that contains just the changes to be made, and applies them to the existing web.xml file to create this file.  That way if there are changes to the base web.xml file in future versions we will not have to manually keep this file up to date.

See what is happening with the server.xml file being modified by the server.xslt file because it is exactly the same as what I am talking about here.

Reply:
Yes that makes sense I went for that method purely because it was the method used in the image I have been looking at for inspiration that you recommended, but I agree that seems like a better way to do that, will make those changes thanks.